content/getting-started/events-api-best-practices/best-practices.md

```diff
@@ -0,0 +1,88 @@
+# Best practices to use the Events API with your connector
+
+<!--
+- Explication rapide de l'events api + lien vers la référence doc de l'events api + helpcenter
+- Explication rapide d'un use-case d'une extension connecté à l'events api
+  - aka, mon site ecommerce aimerait se sync avec le pim rapidement aprés des modifications sur un produit
+  -->
+
+This page presents best practices and recommendations you might need while implementing the Events API inside an API based connector.
+
+First, if you need help to setup a connector to use the Events API take a loot at the [Quick start my first webhook](/getting-started/quick-start-my-first-webhook/welcome.html) guide.
+
+## Configure your server
+
+TODO
+
+<!--
+- recommandations devops (voir avec Pierre-yves)
+  - DNS, Firewall, TLS certificate (+lib TLS)
+  - TLS car la data doit être sécurisé entre le PIM et l'extension
+  - https://api.akeneo.com/events-documentation/limits-and-scalability.html#extension-side-limits
+-->
+
+## Handle API events asynchronously
+
+### Connector scalability
+
+The Events API can send a lot of requests to your connector endpoint. The default value being 4000 requests per hour and each one may contain up to 10 events (see [limits and scalability](/events-documentation/limits-and-scalability.html#limit-of-event-api-requests-per-hour)).
+
+This means that your connector needs to be able to handle up to 40,000 events per hour.
+
+However, each of these events (like `product.updated`) may require additional calls to the PIM Rest API.
+
+For example, you will want to get the attributes and their types for the family of the product, or you will want to fetch more data for the reference entities and assets attributes.
+
+If you do each of this operation synchronously for the 40,000 events, your connector (and the Rest API of the PIM) won't be able to follow and support the load.
+
+### Connector response time
+
+At the same time, your connector needs to acknowledge quickly (under 500ms) that a request sent by the Events API has been handled successfully.
+
+So it excludes, even more, processing the events synchronously upon receiving them.
+
+### Recommendations
+
+We recommend you to process the API events asynchronously.
+
+What it means is that for each Events API request you receive, you should store the events somewhere (to be processed later) and reply immediately to the Events API with a success response.
+
+::: tips
+A typical solution for this would be to store the events in a database or in a [message queue](https://en.wikipedia.org/wiki/Message_queue).
```

Comment:
```suggestion
A typical solution for this would be to store the events in a database or a [message queue](https://en.wikipedia.org/wiki/Message_queue).
```